packages/block-editor/src/components/border-radius-control/index.js

```diff
@@ -50,11 +50,27 @@ export default function BorderRadiusControl( { onChange, values } ) {
 		! hasDefinedValues( values ) || ! hasMixedValues( values )
 	);
 
+	// Tracking selected units via internal state allows filtering of CSS unit
+	// only values from being saved while maintaining preexisting unit selection
+	// behaviour. Filtering CSS only values prevents invalid style values.
```

Comment:
Should this be `Filtering CSS unit only values`?
```suggestion
	// behaviour. Filtering CSS unit only values prevents invalid style values.
```